docs/DeveloperGuide.adoc

```diff
@@ -12,7 +12,13 @@ ifdef::env-github[]
 endif::[]
 :repoURL: https://github.com/CS2103JAN2018-W09-B2/main/tree/master
 
-By: `Team W09-B2`      Since: `Feb 2018`      Licence: `MIT`
+By: `Team W09-B2`      Since: `Mar 2018`      Licence: `MIT`
+
+== Intro
+
+Welcome to contactHeRo developer guide. This guide aims to contain all the information required to quickly get you to start developing contactHeRo. +
```

Comment:
I think this line can be changed to "This guide contains all the information you need to quickly get started in developing contactHeRo"